README.md

```diff
@@ -4,16 +4,30 @@
 
 ## Description
 
-Small wrapper to enable running arbitrary docker run commands in [nomad](https://www.nomadproject.io/).
-This is useful when you require features that are not supported by the nomad docker task driver.
+Small wrapper to enable running arbitrary docker run commands in [Nomad](https://www.nomadproject.io/).
+This is useful when you require features that are not supported by the Nomad docker task driver.
 
 ## Usage
 
-You can use the wrapper script as a job artifact to run your docker container with the specified arguments. Using a named container will ensure that it gets removed properly the next time you start it again. You have to provide `NOMAD_DOCKER_CONTAINER_NAME` as an environment variable.
+You can use the wrapper script as a job artifact to run your docker container with the specified arguments.
+
+## Docker Container Name
+
+Using a named container will ensure that it gets removed properly the next time you start it again. You have to provide `NOMAD_DOCKER_CONTAINER_NAME` as an environment variable.
+
+## Environment Variable Replacement
+
+If you want to use environment variables inside your docker command that are passed by Nomad (e.g. dynamic ports), you can include them as a String and rely on the wrapper to replace them. To accomplish this, just set `NOMAD_DOCKER_ENVSUBST` to any value.
+
+*Please make sure that `envsubst` is installed on your system when using this option. It is part of the `gettext` package: `apt-get install gettext`.*

```

Comment:
In your machine deployment (e.g. through ansible), you would install gettext alongside with docker and nomad.